ddtrace/internal/_encoding.pyx

```diff
@@ -244,6 +244,11 @@ cdef class MsgpackStringTable(StringTable):
     cdef insert(self, object string):
         cdef int ret
 
+        if len(string) > self.max_size:
+            string = "<dropped string of length %d in ddtrace.internal._encoding.pyx due to overrun of buffer of length %d>" % (
```

Comment:
This is what users will see in the UI as the value of a tag that has a really long value? I am not sure including `ddtrace.internal._encoding.pyx` will mean much to users.

For my own purposes, this is to drop strings that themselves are larger than the total buffer size?

Reply:
That's right @brettlangdon